spec/models/collection_spec.rb

```diff
@@ -44,7 +46,17 @@
         allow(@collection).to receive(:members).and_return([file, file, file])
         allow(ActiveFedora::SolrService).to receive(:query).with(query, args).and_return(documents)
       end
-      it { is_expected.to eq 99 }
+
+      context "saved" do
+        before do
+          allow(@collection).to receive(:new_record?).and_return(false)
+        end
+        it { is_expected.to eq 99 }
+      end
+
+      context "not saved" do
+        it { expect {subject}.to raise_error RuntimeError }
```

Comment:
Can you also add the "Record must be saved to query for bytes" string here to make sure you're catching the right `RuntimeError`?